crates/bevy_ui/src/lib.rs

```diff
@@ -50,7 +50,11 @@ use update::update_clipping_system;
 
 /// The basic plugin for Bevy UI
 #[derive(Default)]
-pub struct UiPlugin;
+pub struct UiPlugin {
+    /// When set to true, UI rendering won't be added to the `RenderApp` and the UI will not be shown.
+    /// The UI layout and interaction components will still be updated as normal.
+    pub disable_ui_rendering: bool,
```

Comment:
I think having a boolean in the negative is a bit weird, semantically. Could you change this to `ui_rendering: bool` with a default of `true`?